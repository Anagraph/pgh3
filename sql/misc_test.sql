create extension if not exists postgis;
create extension if not exists pgh3;

/******* misc functions *********************************/

select h3_hexagon_area_km2(4);

select h3_hexagon_area_m2(4);

select h3_edge_length_km(4);

select h3_edge_length_m(4);
