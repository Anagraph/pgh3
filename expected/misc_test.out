create extension if not exists postgis;
NOTICE:  extension "postgis" already exists, skipping
create extension if not exists pgh3;
NOTICE:  extension "pgh3" already exists, skipping
/******* misc functions *********************************/
select h3_hexagon_area_km2(4);
 h3_hexagon_area_km2 
---------------------
         1770.323552
(1 row)

select h3_hexagon_area_m2(4);
 h3_hexagon_area_m2 
--------------------
         1770323552
(1 row)

select h3_edge_length_km(4);
 h3_edge_length_km 
-------------------
        22.6063794
(1 row)

select h3_edge_length_m(4);
 h3_edge_length_m 
------------------
       22606.3794
(1 row)

